src/domains/charts/getChartMenu.js

```diff
@@ -57,6 +56,13 @@ export default (
     case "anomaly":
       return emit({})
 
+    case "disk":
+    case "mount": {
+      if (!part2) return emit({ menu: part1 })
+      if (part2 === "inodes" || part2 === "space") return emit({ menu: "mount" })
+      return emit({ menu: part1 })
```

Comment:
Why not `if (part2 && ((part2 === "inodes" || part2 === "space"))) return emit({ menu: "mount" })`?
Just to avoid having the `return emit({ menu: part1 })` twice! 😄

Reply:
We can just erase the whole ` if (!part2) return emit({ menu: part1 })` I think 🤔